pandas/tests/extension/test_numpy.py

```diff
@@ -374,19 +374,6 @@ def test_setitem_mask(self, data, mask, box_in_series):
     def test_setitem_integer_array(self, data, idx, box_in_series):
         super().test_setitem_integer_array(data, idx, box_in_series)
 
-    @pytest.mark.parametrize(
-        "idx, box_in_series",
-        [
-            ([0, 1, 2, pd.NA], False),
-            pytest.param([0, 1, 2, pd.NA], True, marks=pytest.mark.xfail),
-            (pd.array([0, 1, 2, pd.NA], dtype="Int64"), False),
-            (pd.array([0, 1, 2, pd.NA], dtype="Int64"), False),
-        ],
-        ids=["list-False", "list-True", "integer-array-False", "integer-array-True"],
-    )
-    def test_setitem_integer_with_missing_raises(self, data, idx, box_in_series):
```

Comment:
Do you know why this test got deleted?

Reply:
The test definition here is essentially the same as the base test